iii_account/iii_account.py

```diff
@@ -129,6 +129,8 @@ def get_items(self, bib):
             _k['location'] = loc
             _k['callnumber'] = call
             _k['status'] = status
+            _k['barcode'] = barcode
+            _k['barcode_contiguous'] = barcode.replace( u' ', u'' )
```

Comment:
Can we just have one barcode?  Stripping the spaces seems fine if the server accepts (which I think it will).  Don't want to have two barcodes adds clutter and ambiguity for something that is a unique identifier.